src/rln.ts

```diff
@@ -26,6 +26,20 @@ function concatenate(...input: Uint8Array[]): Uint8Array {
   return result;
 }
 
+/**
+ * Transforms Uint8Array into BigInt
+ * @param array: Uint8Array
+ * @returns BigInt
+ */
+function buildBigIntFromUint8Array(array: Uint8Array): bigint {
+  const bigEndianArray = array.slice().reverse();
+  // hack for Uint8Array.map that has Uint8Array -> Uint8Array definition that prevents from mapping to other types
+  const hexString = (bigEndianArray as unknown as number[])
```

Comment:
As mentioned in the comment it is a hack to overcame default types for `Uint8Array.map` method.
cc: @fryorcraken

Reply:
Why not simply using https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/DataView/getBigUint64 ?

Reply:
indeed...